2024/2024-ssl-passingAndShooting-rules.adoc

```diff
@@ -0,0 +1,90 @@
+:source-highlighter: highlightjs
+
+= RoboCup Brasil 2024 SSL Passing and Shooting
+{docdate}
+:toc:
+:stem: latexmath
+:sectnumlevels: 0
+
+// add icons from fontawesome in a up-to-date version
+ifdef::backend-html5[]
+++++
+<link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.3.1/css/all.css" integrity="sha384-mzrmE5qonljUremFsqc01SB46JvROS7bZs3IO2EmfFsd15uHvIt+Y8vEf7N7fWAU" crossorigin="anonymous">
+++++
+endif::backend-html5[]
+
+:icons: font
+:numbered:
+
+NOTE: References to the male gender in the rules with respect to referees, team
+members, officials, etc. are for simplification and apply to both males and
+females.
+
+== Goals of the Technical Challenge
+
+In this challenge, the central objective is to test the ability of a set of attackers to score a goal by making at least one pass to an allied robot and finishing the play with a shot at the opponent's goal.
+
+Within a limited timeframe, the team needs to make as many successful passes as possible, culminating in a shot at the opponent's goal to complete the play. This challenge not only assesses the participants' technical skills but also their ability to work as a team and make quick, practical decisions during the match.
+
+By focusing on the precise execution of passes, collaboration between robots, and finishing the play with a shot at the opponent's goal, this challenge encourages innovation and continuous development in the SSL - Regular League.
+
+=== Participation Requirements
+
+All teams of the Regular League are eligible and *are encouraged* to participate in this challenge.
+
+== Procedure
+There will be five different situations in random order that teams have to handle. The situations include the positions of the opposing robots and the ball. Put up to robots on the field and score as many goals as possible within 300 seconds.
+
+* To start, all robots must be placed on the middle line.
```

Comment:
Regardless of the scenario, the robots must be positioned in the middle of the field so that all teams start the challenge under the same conditions, as time is a tiebreaker.

Yes, I wanted to avoid shots on goal in scenarios where the ball will be positioned in the defensive field.

Reply:
Ok.